Dockerfile

```diff
@@ -9,10 +12,13 @@ ENV BUNDLE_PATH="/bundle_cache"\
         BUNDLE_APP_CONFIG="/bundle_cache"\
         GEM_HOME="/bundle_cache"\
         PATH=/bundle_cache/bin:/bundle_cache/gems/bin:$PATH\
-        PORT=2999
+        PORT=2999\
+        DEBUGGER=${DEBUGGER}
```

Comment:
This environment variable used in the Gemfile to determine which debugger gems to install.